mods/cnc/chrome/missionbrowser.yaml

```diff
@@ -68,6 +68,44 @@ Container@MISSIONBROWSER_PANEL:
 									IgnoreMouseOver: True
 									IgnoreMouseInput: True
 									ShowSpawnPoints: False
+						Container@MISSION_MINI_OPTIONS:
+							Width: PARENT_WIDTH
+							Height: PARENT_HEIGHT
+							Visible: False
+							Children:
+								LabelForInput@DIFFICULTY_DESC:
+									Y: PARENT_HEIGHT - HEIGHT + 2
+									Width: 56
+									Height: 21
+									Align: Right
+									Text: dropdown-missionbrowser-difficulty.label
+									For: DIFFICULTY
+								DropDownButton@DIFFICULTY:
+									X: 61
+									Y: PARENT_HEIGHT - HEIGHT + 2
+									Width: 120
+									Height: 21
```

Comment:
IMO these narrower buttons look out of place. Can we go with the standard 25px height?

Reply:
Sure, I had reverted to the original size